extensions/machine-learning-services/images/ML_mlflow_DisableMLFlow.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Can you please add a snip/gif of the overall look in description

Reply:
deleted for now. will be in next PR

---
In reply to: [356334015](https://github.com/microsoft/azuredatastudio/pull/8622#discussion_r356334015) [](ancestors = 356334015)